ocaml/xapi/message_forwarding.ml

```diff
@@ -2578,6 +2578,42 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
         (fun session_id rpc ->
            Client.Host.get_server_certificate rpc session_id host)
 
+    let install_server_certificate ~__context ~host ~certificate ~private_key ~certificate_chain =
+      let task = Context.get_task_id __context in
+      let local_fn =
+        Local.Host.install_server_certificate
+          ~host
+          ~certificate
+          ~private_key
+          ~certificate_chain
+      in
+      let success () =
+        let progress = Db.Task.get_progress ~__context ~self:task in
+        if progress = 1. then Some () else None
```

Comment:
You might want to make this `if progress >= 1.0` because you are dealing with `float`

Reply:
Is that the case even if the value is set explicitly to 1.0, that is, no arithmetics are used to set `progress`?